descartes_light/core/include/descartes_light/core/waypoint_sampler.h

```diff
@@ -60,7 +60,13 @@ class WaypointSampler
    */
   virtual typename std::vector<StateSample<FloatType>> sample() const = 0;
 
-  friend std::ostream& operator<<(std::ostream& os, const WaypointSampler<FloatType>& /*wps*/) { return os; }
+  virtual void print(std::ostream& os) const {os << "";}
```

Comment:
Let's make the default behavior do nothing

```suggestion
  virtual void print(std::ostream&) const { }
```